lib/api_client.js

```diff
@@ -2129,6 +2215,40 @@ APIClient.prototype.sendTransaction = function(identifier, txHex, paths, checkFe
     );
 };
 
+APIClient.prototype.sendRskTransaction = function(identifier, txHex, txid, checkFee, twoFactorToken, prioboost, options, cb) {
+    var self = this;
+
+    if (typeof twoFactorToken === "function") {
+        cb = twoFactorToken;
+        twoFactorToken = null;
+        prioboost = false;
+    } else if (typeof prioboost === "function") {
+        cb = prioboost;
+        prioboost = false;
+    } else if (typeof options === "function") {
+        cb = options;
+        options = {};
+    }
+
+    var data = {
+        two_factor_token: twoFactorToken,
+        raw_transaction: txHex,
+        transaction_id: txid
```

Comment:
naming nit re transaction_id. was this required for the btc style txs? might be a legacy thing (reviewing nodejs sdk first though)

Reply:
It's not required for broadcasting. But backend does return txid in a view.